example_test.go

```diff
@@ -311,3 +341,27 @@ func Example_copyArtifactManifestRemoteToLocal() {
 	// Output:
 	// true
 }
+
+// Example_extendedCopyArtifactManifestRemoteToLocal gives an example of
+// copying an artifact manifest, its predecessors and successors from a remote
+// repository to local.
+func Example_extendedCopyArtifactManifestRemoteToLocal() {
+	src, err := remote.NewRepository(fmt.Sprintf("%s/source", remoteHost))
+	if err != nil {
+		panic(err)
+	}
+	dst := memory.New()
+	ctx := context.Background()
+
+	tagName := "latest"
+	// ExtendedCopy will copy all reachable nodes from src (i.e. its
+	// predecessors and successors).
```

Comment:
Let's avoid using the term `predecessors` and `successors` here, as it might be hard for users to understand.
How about: copy the artifact tagged by "latest" along with all of its referrers from src to dst.

Reply:
Changed accordingly.